tests/collection/collection-layout.spec.ts

```diff
@@ -0,0 +1,157 @@
+import { test } from '../fixtures';
+
+import { LayoutViewModeLocator, SearchOption } from '../models';
+
+test.beforeEach(async ({ collectionPage }) => {
+  test.info().annotations.push({
+    type: 'Test',
+    description: 'Collection MDS search every each test',
+  });
+
+  await test.step(`Select "Search metadata"`, async () => {
+    await collectionPage.searchPage.clickSearchInputOption(
+      SearchOption.METADATA,
+    );
+  });
+
+  await test.step(`Query for "radio"`, async () => {
+    await collectionPage.searchPage.queryFor('radio');
+  });
+
+  await test.step(`Total result count is displayed`, async () => {
+    await collectionPage.collectionFacets.checkResultCount();
+  });
+});
+
+test('Tile, List, and Compact layout buttons change layout', async ({
+  collectionPage,
+}) => {
+  await test.step('Click list view mode and check if it displays correctly', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.LIST,
+    );
+    await collectionPage.infiniteScroller.assertLayoutViewModeChange('list');
+  });
+
+  await test.step('Click compact view mode and check if it displays correctly', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.COMPACT,
+    );
+    await collectionPage.infiniteScroller.assertLayoutViewModeChange('compact');
+  });
+
+  await test.step('Click tile view mode and check if it displays correctly', async () => {
+    await collectionPage.infiniteScroller.clickViewMode(
+      LayoutViewModeLocator.TILE,
+    );
+    await collectionPage.infiniteScroller.assertLayoutViewModeChange('tile');
+  });
+});
+
+test('Tile hover pane appears', async ({ collectionPage }) => {
+  await test.step('Hover first item tile and check for title', async () => {
+    await collectionPage.infiniteScroller.hoverToFirstItem();
+  });
+
+  await test.step('Check title text inside tile-hover-pane and item-tile', async () => {
+    await collectionPage.infiniteScroller.assertTileHoverPaneTitleIsSameWithItemTile();
```

Comment:
suggestion: can we roll this up to 1 test step? when we are hovering in the code?